### ISSUE TITLE
amp-wp.org links is not helpful for end uses and is bad for SEO

### ISSUE THREAD

**Issue by raja-anbazhagan** [Automattic/amp-wp#6076]
## Bug Description

Check this screen shot.

![AMP error help](https://user-images.githubusercontent.com/8947215/115119658-2439ad00-9fc7-11eb-8878-90d57a18726b.png)

This error message contains a link to https://amp-wp.org/?p=5463 Which I believe no end user would need or want to see. 

Also It is also not good for a healthy SEO. 

## Expected Behaviour

No links that are irrelevant to the site should be visible for end user.

## Steps to reproduce

1. Open Chrome and navigate to "https://springhow.com" and open any recent post
2. Open Chrome Developer tools and Go to network tab
3. Add a Network request blocking for the pattern "https://springhow.com/wp-comments-post.php*"
4. Now submit a comment And you will see an error as shown in the screenshot.

## Additional context

 - WordPress version: 5.7.1
 - Plugin version: 2.0.11
 - Gutenberg plugin version (if applicable):
 - AMP plugin template mode: standard
 - PHP version: 7.3
 - OS: Debian
 - Browser: Chrome
 - Device: Windows

---------------

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by westonruter** [comment]
I don't know if it would negatively impact your SEO, but we should add `rel="nofollow noreferrer noopener"` to the link.

**Comment by westonruter** [comment]
Aside: The error message shown in your screenshot should also be much improved once we get to implement https://github.com/ampproject/amp-wp/issues/4191.

**Comment by westonruter** [comment]
> This error message contains a link to https://amp-wp.org/?p=5463 Which I believe no end user would need or want to see.

As noted https://github.com/ampproject/amp-wp/issues/6076#issuecomment-821864402, the message will be improved with #4191. But additionally, this message will be shown less frequently entirely due to better handling of redirects. Otherwise, the message is currently there to encourage developers to better handle form submissions in their plugin code.

The SEO concern will be addressed by #6082.

For the rest, please follow #4191.

**Comment by westonruter** [comment]
**QA Passed**

I added a Custom HTML block as follows:

```html
<!-- wp:html -->
<form method=post action="/not-found/">
<input type=text name=name>
<button type=submit>Submit</button>
</form>
<!-- /wp:html -->
```

On an AMP page, submitting the form results in:

![image](https://user-images.githubusercontent.com/134745/115981780-2feb1d80-a54b-11eb-8e33-77a1fcce5abf.png)

The link has `nofollow`, `noreferrer`, `noopener`:

![image](https://user-images.githubusercontent.com/134745/115981786-485b3800-a54b-11eb-821f-533769e4fa4f.png)
